templates/mainpage-template.html

```diff
@@ -0,0 +1,249 @@
+	<link rel="stylesheet" href="http://www.jazzhistorydatabase.com/scripts/canvas/HTML/css/dark.css" type="text/css"/>
+	<link rel="stylesheet" href="http://www.jazzhistorydatabase.com/scripts/canvas/HTML/css/font-icons.css" type="text/css"/>
+	<link rel="stylesheet" href="http://www.jazzhistorydatabase.com/scripts/canvas/HTML/css/animate.css" type="text/css"/>
+	<link rel="stylesheet" href="http://www.jazzhistorydatabase.com/scripts/canvas/HTML/css/magnific-popup.css" type="text/css"/>
+	<link rel="stylesheet" href="http://www.jazzhistorydatabase.com/scripts/canvas/HTML/css/responsive.css" type="text/css"/>
+	<link rel="stylesheet" href="http://www.jazzhistorydatabase.com/scripts/canvas/HTML/css/JHDB.css" type="text/css"/>
+
+	<meta name="viewport" content="width=device-width, initial-scale=1"/>
+
+	<!-- Document Title
+	============================================= -->
+	<title>Jazz History Database</title>
+
+</head>
+
+<body>
+
+	<!-- Document Wrapper
+	============================================= -->
+
+	<div id="wrapper" class="clearfix">
+
+
+
+		<!-- Header
+		============================================= -->
+
+		{{header}}
+
+		<!-- Page Title
+		============================================= -->
+		<!-- Intro -->
+		<style>
+			#page-title {
+				padding: 0 0;
+			}
+			
+			img.resize {
+				width: auto;
+				/* you can use % */
+				height: 300px;
+			}
+			
+			a.resize {
+				max-height: 300px;
+				height: 300px;
+			}
+			
+			li>a {
+				cursor: pointer;
+			}
+		</style>
+
+
+		<!-- Content
+		============================================= -->
+		<section id="content">
+
+			<br>
+
+			<!-- Collection nav bar start-->
+			<div id="page-menu">
+				<div id="page-menu-wrap">
+					<div class="container clearfix">
+						<div class="menu-title" style="cursor:pointer;" onclick="goToByScroll('section-bio')"> Name Collection </div>
+						<nav>
+							<ul>
+								<li>
+									<a onclick="goToByScroll('bio')">
+										<div>Biography</div>
+									</a>
+								</li>
+
+								<li>
+									<a onClick="goToByScroll('images')">
+										<div>Images</div>
+
+									</a>
+								</li>
+							</ul>
+						</nav>
+						<div id="page-submenu-trigger"><i class="icon-reorder"></i>
+						</div>
+					</div>
+				</div>
+			</div>
+			<!-- Collection nav bar end -->
+			<!-- Content Wrap starts -->
+			<div class="content-wrap">
+				<!-- Bio -->
+				<div class="container clearfix">
+					<div class="fancy-title title-bottom-border" id="bio">
+						<h2>Biography of Name</h2>
+					</div>
+					<!-- Insert bio -->
+					<div class="coll-main-bio">
+						<div class="col_three_fifth bottommargin-sm">
+							<p class="bio">
+								text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text text
```

Comment:
Dang...this could have been our opportunity to use lorem ipsum. I know no one will see this, but man